scripts/release.sh

```diff
@@ -42,7 +42,7 @@ done
 #strip                                     target/install/bin/*
 
 # 2) cita
-cp -rf scripts/cita                        target/install/bin/
+cp -rf scripts/cita.sh                      target/install/bin/cita
 
```

Comment:
`cp -rf scripts/cita.sh                      target/install/bin/cita.sh` ?

Reply:
Np.
Rename `cita` to `cita.sh` for using shell-check.
But still use `bin/cita usage`.

Reply:
OK, I know